pom.properties

```diff
@@ -0,0 +1,4 @@
+artifactId=${project.artifactId}
+groupId=${project.groupId}
+version=${project.version}
+build-time=${maven.build.timestamp}
```

Comment:
I guess this should use camelCase, same as the other ones?

Reply:
You're right. Fixed. :)

Reply:
Perfect, thanks!